### PR TITLE
add help links

### PR DESCRIPTION
**Pull request by makella**
By chance, I was looking at the Help Center last Friday and realized that there are a lot of words that we are using in the Glossary section there. I went through and added some of the following definitions throughout the Guides:

```
zoom:
https://carto.com/help/glossary/#basemap
https://carto.com/help/glossary/#zoomlevels

expression:
https://carto.com/help/glossary/#expression
https://carto.com/help/glossary/#visualization

sources:
https://carto.com/help/glossary/#geojson
https://carto.com/help/glossary/#query

data driven 2
https://carto.com/help/glossary/#proportionalsymbols
https://carto.com/help/glossary/#colorscheme
https://carto.com/help/glossary/#colorbrewer


data driven 1
https://carto.com/help/glossary/#classificationmethods
https://carto.com/help/glossary/#equalintervals
https://carto.com/help/glossary/#quantiles

interactivity:
https://carto.com/help/glossary/#interactivity
https://carto.com/help/glossary/#variable

https://carto.com/help/glossary/#datatype
https://carto.com/help/glossary/#dataset
```

### CODE REVIEW

File: docs/guides/07-zoom-based-styles.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ## Zoom-based styles
-An inherent feature of any map viewed on the web is the ability to zoom in and out. Each time you zoom in or out of a map, you are viewing a different _zoom level_. At each zoom level or range of zoom levels, there are important design considerations for how data are visualized and/or what information is displayed.
+An inherent feature of any map viewed on the web is the ability to zoom in and out. Each time you zoom in or out of a map, you are viewing a different [_zoom level_](https://carto.com/help/glossary/#zoomlevels). At each zoom level or range of zoom levels, there are important design considerations for how data are visualized and/or what information is displayed.
```
Comment (VictorVelarde) [inline docs/guides/07-zoom-based-styles.md]: I think we can remove the underscores now that it is a link.

File: docs/guides/09-add-interactivity-and-events.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ## Add interactivity and events
-In this guide you will learn how to add user interactions to your CARTO VL visualization. By the end of this guide, you will have a better understanding of interactivity events (e.g. waiting for a layer to load or clicking on a feature) and how they can be used to make your visualization more dynamic and provide a richer experience for the end-user (e.g. changing the color of features on hover or click). You will also learn how to add common add-ons like *pop-ups* with additional information about the data being clicked on or hovered.
+In this guide you will learn how to add user interactions to your CARTO VL visualization. By the end of this guide, you will have a better understanding of [interactivity](https://carto.com/help/glossary/#interactivity) events (e.g. waiting for a layer to load or clicking on a feature) and how they can be used to make your visualization more dynamic and provide a richer experience for the end-user (e.g. changing the color of features on hover or click). You will also learn how to add common add-ons like *pop-ups* with additional information about the data being clicked on or hovered.
```
Comment (VictorVelarde) [inline docs/guides/09-add-interactivity-and-events.md]: I think that in this context it is better to rephrase it a bit (the old one was a little confusing too), as "waiting for a layer" is an event, but not an interactivity (as per the glossary definition).


```suggestion
In this guide you will learn how to add user interactions to your CARTO VL visualization. By the end of this guide, you will have a better understanding of events (e.g. waiting for a layer to load or clicking on a feature) and how they can be used to make your visualization more dynamic and also provide a richer experience for the end-user (e.g. changing the color of features on hover or click, thanks to our [interactivity](https://carto.com/help/glossary/#interactivity) features). You will also learn how to add common add-ons like *pop-ups* with additional information about the data being clicked on or hovered.
```

File: docs/guides/10-add-legends.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ## Add legends
 
-Maps that symbolize data without the necessary information to decode the symbols are not always effective in communicating their message. In this guide, you will explore how to enrich your visualizations with **legends** to aid interpretation by providing a visual explanation of point, line, or polygon symbols used on a map with a description of what they represent.
+Maps that symbolize data without the necessary information to decode the symbols are not always effective in communicating their message. In this guide, you will explore how to enrich your visualizations with [**legends**](https://carto.com/help/glossary/#legend) to aid interpretation by providing a visual explanation of point, line, or polygon symbols used on a map with a description of what they represent.
```
Comment (VictorVelarde) [inline docs/guides/10-add-legends.md]: I would remove the bold now that it is a link

File: docs/guides/10-add-legends.md [hunk 4]
```diff
@@ -121,7 +121,7 @@ const viz = new carto.Viz(`
 Since the `ramp` expression is the root expression of a styling property (in this case, `color`) it can be accessed directly with `layer.viz.color.getLegendData()`.
 
 **Note:**
-If styling isn't tied directly to a styling property, you will need to use a variable. We will explore that method in detail in the [Add Widgets](/developers/carto-vl/guides/add-widgets/) guide.
+If styling isn't tied directly to a styling property, you will need to use a [variable](https://carto.com/help/glossary/#variable). We will explore that method in detail in the [Add Widgets](/developers/carto-vl/guides/add-widgets/) guide.
```
Comment (VictorVelarde) [inline docs/guides/10-add-legends.md]: I think we could use the more precise definition from CARTO VL and not the general glossary definition on this here. What do you think?

File: docs/guides/11-add-widgets.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
 
-The first thing you need to know to create widgets is the concept of **variables**. A variable is a name in a CARTO VL visualization that has an expression bound to it. It's simply a way to name expressions to be referenced later. For example, let's say there was a magic function that returned our widget information like `myWidgetExpression()`, how do we actually get that information?
+The first thing you need to know to create widgets is the concept of [**variables**](https://carto.com/help/glossary/#variable). A variable is a name in a CARTO VL visualization that has an expression bound to it. It's simply a way to name expressions to be referenced later. For example, let's say there was a magic function that returned our widget information like `myWidgetExpression()`, how do we actually get that information?
```
Comment (VictorVelarde) [inline docs/guides/11-add-widgets.md]: Same here. In fact, I'm surprised there is a variable term in the glossary talking just about CARTO VL

Comment (makella) [reply docs/guides/11-add-widgets.md]: yep! that's what i realized about a lot of the glossary terms... they are actually for CARTO VL!

File: docs/guides/10-add-legends.md (hunk 4 above)
Comment (makella) [reply docs/guides/10-add-legends.md]: ahhh ok, yes. missed that we had that in our glossary.

File: docs/guides/09-add-interactivity-and-events.md (hunk 2 above)
Comment (makella) [reply docs/guides/09-add-interactivity-and-events.md]: 👍

**Comment by VictorVelarde** [comment]
Hi @makella, can you please check this one to merge it later on?

**Comment by makella** [comment]
yes! i will fix these things.

**Comment by makella** [comment]
@VictorVelarde included your review suggestions + fixed the broken links reported by @ramiroaznar (https://github.com/CartoDB/developers/issues/481).

thanks!

File: docs/guides/04-data-driven-visualizations-part-1.md [hunk 6]
```diff
@@ -46,7 +46,7 @@ color: ramp($population_density, [midnightblue, deeppink, gold])
 ```
 #### Implicit casts
 
-Matching the input with the context of the lowest population density and highest population density is done by the [`linear`](/developers/carto-vl/reference/#cartoexpressionslinear) function, which is used automatically by `ramp` when the input is a numeric property. This means that the CARTO VL `ramp` function makes transformations that we call *implicit casts*.
+Matching the input with the context of the lowest population density and highest population density is done by the [`linear`](/developers/carto-vl/reference/#cartoexpressionslinear) function, which is used automatically by `ramp` when the input is a numeric property. This means that the CARTO VL `ramp` function makes transformations that we call [implicit casts](developers/carto-vl/guides/Glossary/#implicit-cast).
```
Comment (VictorVelarde) [inline docs/guides/04-data-driven-visualizations-part-1.md]: It looks like we're missing the initial "/" for implicit casts

File: docs/guides/11-add-widgets.md [hunk 7]
```diff
@@ -30,13 +30,13 @@ If, however, `myWidgetExpression` was static, we could just use `myLayer.on('loa
 
 #### What is the average price in the entire dataset?
 
-To get the average of a property in the entire dataset we'll need to use the [`globalAvg()`](/carto-vl/reference/#cartoexpressionsglobalavg) expression.
+To get the average of a property in the entire dataset we'll need to use the [`globalAvg()`](/developers/carto-vl/reference/#cartoexpressionsglobalavg) expression.
```
Comment (VictorVelarde) [inline docs/guides/11-add-widgets.md]: 👍 thx Mamata!

File: docs/guides/04-data-driven-visualizations-part-1.md (hunk 6 above)
Comment (VictorVelarde) [reply docs/guides/04-data-driven-visualizations-part-1.md]: That detail and then we're ready to merge!

Comment (makella) [reply docs/guides/04-data-driven-visualizations-part-1.md]: oops! ok, fixing real quick!

File: docs/guides/11-add-widgets.md (hunk 7 above)
Comment (makella) [reply docs/guides/11-add-widgets.md]: no prob!

**Comment by makella** [comment]
@VictorVelarde ok, fixed that detail (thanks for pointing out!!)

**Pull request merged**